Utilities/CreateServicePrincipal.ps1

```diff
@@ -223,16 +223,34 @@ function New-ServicePrincipal() {
     Write-Prompt "2) Owner"
     Write-Prompt "Please choose by entering 1 or 2:"
     $Privilege = Read-Host
-
     Write-Host "Assign roles to Service Principal..."
-    if ($Privilege -eq 2) {
-        New-AzRoleAssignment -RoleDefinitionName "Owner" -ApplicationId $ClientId
-    }
-    else {
-        New-AzRoleAssignment -RoleDefinitionName "Contributor" -ApplicationId $ClientId
+    try {
+        if ($Privilege -eq 2) {
+            New-AzRoleAssignment -RoleDefinitionName "Owner" -ApplicationId $ClientId
+        }
+        else {
+            New-AzRoleAssignment -RoleDefinitionName "Contributor" -ApplicationId $ClientId
+        }
+        Write-Host "Successfully created Service Principal and assigned role...`n"
+    } catch {
+        $ErrorMessage = $_.Exception.Message
```

Comment:
We have common rule printing out the error message with the error line number. Please refer to another place of 'catch block.